openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala

```diff
@@ -104,6 +110,17 @@ object Operation {
 
 }
 
+case class OperationSecurity(name: String,
```

Comment:
What do you think of naming this class `SecurityRequirement` instead, to follow the naming convention of the OAS?

Reply:
Good idea 👍